core/spoofax.compiler/src/main/resources/mb/spoofax/compiler/adapter/adapter_project/Module.java.mustache

```diff
@@ -142,6 +147,19 @@ public class {{baseModule.id}} {
         return strategoRuntime;
     }
 {{/strategoRuntime}}
+{{#tegoRuntime}}
+
+
+    @Provides @{{scope.id}}
+    static mb.tego.strategies.runtime.TegoRuntime provideTegoRuntime(mb.tego.strategies.runtime.TegoRuntimeImpl tegoImplementation) {
+        return tegoImplementation;
+    }
+
+    @Provides @{{scope.id}}
+    static mb.tego.strategies.runtime.TegoEngine provideTegoEngine(mb.tego.strategies.runtime.TegoRuntimeImpl tegoImplementation) {
+        return tegoImplementation;
```

Comment:
The `TegoEngine` interface says that it should be called by Tego strategies. Does it make sense to expose this interface to the entire language? I'd think that the runtime exposes itself as an engine to strategies instead, so there is no need to expose it here, but I don't know the internals enough.